Utilizr.Win32/Advapi32/Advapi32.cs

```diff
@@ -81,5 +81,28 @@ public static extern bool CreateProcessAsUser(
             ref STARTUPINFO lpStartupInfo,
             out PROCESS_INFORMATION lpProcessInformation
         );
+
+        [DllImport(ADVAPI32_DLL, CharSet = CharSet.Auto, SetLastError = true)]
+        public static extern IntPtr OpenSCManager(string machineName, string databaseName, uint dwAccess);
+
+        [DllImport(ADVAPI32_DLL, CharSet = CharSet.Auto, SetLastError = true)]
+        public static extern IntPtr OpenService(IntPtr hSCManager, string lpServiceName, uint dwDesiredAccess);
+
+        [DllImport(ADVAPI32_DLL, CharSet = CharSet.Auto, SetLastError = true)]
+        public static extern bool ChangeServiceConfig(
+            IntPtr hService,
+            uint nServiceType,
+            uint nStartType,
+            uint nErrorControl,
+            string lpBinaryPathName,
+            string lpLoadOrderGroup,
+            IntPtr lpdwTagId,
+            [In] char[] lpDependencies,
+            string lpServiceStartName,
+            string lpPassword,
+            string lpDisplayName);
```

Comment:
If any of these `string` parameters can be null, we can use `string?` to prevent all the warnings.

![image](https://github.com/user-attachments/assets/64e58217-8228-41fa-b974-9384a9459224)